uefi/uefi_test.go

```diff
@@ -30,3 +31,136 @@ func TestUnmarshalTypedFirmware(t *testing.T) {
 		t.Errorf("got %q, expected CHARLIE", outSection.Name)
 	}
 }
+
+var (
+	// Checksum Tests
+	emptyBuf  = []byte{}
+	sampleBuf = []byte{1, 2, 3, 4}
+	overBuf   = []byte{0x1, 0x2, 0xFF, 0xFF}
+	zeroBuf   = []byte{0, 0, 0, 0}
+	threeBuf  = []byte{3, 3, 3}
+)
+
+func TestChecksum8(t *testing.T) {
+	var tests = []struct {
+		name string
+		buf  []byte
+		res  uint8
+	}{
+		{"emptyBuf", emptyBuf, 0},
+		{"sampleBuf", sampleBuf, 10},
+		{"overBuf", overBuf, 0x1},
+		{"zeroBuf", zeroBuf, 0},
+		{"threeBuf", threeBuf, 9},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
Remove `t.Parallel()`. It is creating false positives.

Reply:
Done